enterprise/server/remote_execution/copy_on_write/copy_on_write.go

```diff
@@ -1368,9 +1371,15 @@ func (ml *MmapLRU) Close() error {
 }
 
 func (ml *MmapLRU) processEvictions() error {
+	var eg errgroup.Group
+	eg.SetLimit(lruEvictionConcurrency)
 	for m := range ml.evictions {
-		ml.processEviction(m)
+		eg.Go(func() error {
+			ml.processEviction(m)
+			return nil
+		})
 	}
+	eg.Wait()
 	return nil
 }
```

Comment:
Starting a goroutine for every eviction seems expensive - I think instead, we want to do this on line 128:

```
for range lruConcurrency {
	ml.evictorGroup.Go(ml.processEvictions)
}
```

then each `processEvictions` worker will pop values off the same channel and stop when it's closed

Reply:
I wasn't sure if it was safe for 2 goroutines to concurrently range over a channel, but looks like that's okay